.eslintrc.js

```diff
@@ -0,0 +1,48 @@
+module.exports = {
```

Comment:
You kept both `.eslintrc.js` and `.eslintrc.json` in the codebase. Did you just forget to delete `.eslintrc.json`?

Reply:
As I mentioned, this PR is still a work in progress (I didn't expect to get feedback so quickly). I only opened this PR to keep a visible track on it, but I haven't even taken the time myself to review and cleanup my work 😅 
However, thank you for your feedback. This is always very useful.